### DISCUSSION TITLE
Multiplied releases

### DISCUSSION THREAD

**Discussion by jed84** [valinet/ExplorerPatcher#412] [General]
Is there a specific reason why are we getting multiple of the same (similar?) releases within minutes?
Just today we have got 22000.318.38.0, 22000.318.38.1 and 22000.318.38.2 and this is happening most of the time when there is a new release.

I appreciate the new features and the hard work of the dev.

I just wonder if there is a specific reason why releases are getting multiplied?

**Comment by valinet** [comment]
First of all, please, issues is for issues. Is this an issue? I don't think so. Then post in discussions, please.

Second of all, how should I push builds then? Delay new features for a few hours just so, or how? it just happened that I implemented something, and then something else, so naturally I released that as well. Builds are continuously generated for releases, which just happen sometime one after the other in quick succession. Reason? Idk, personal inspiration, hard work, patience, maybe. I usually pack multiple things together, but sometimes it just happens I discover something after pushing a release. That's all.

**Comment by jed84** [comment]
To be fair I dont think it's a good idea:
![obraz](https://user-images.githubusercontent.com/47121522/142488365-d362dbb4-1291-45c4-af59-4c96a482f44e.png)
This is just from today. Imagine getting this by email + within OS notifications. You update an app, you get a noticiation by email+OS, you have to restart your OS, and within 5 minutes there is another update to install. And another one. And another one.

I believe 1 release per 1-2 weeks is enough, there is no need to rush.
This is nothing against your hard work. It's much appreciated! More about handling your hard work to the users ^^

Perhaps a beta channel for minor releases would be a good solution?

**Reply by Anixx** [reply]
If you dislike frequent updates, just do not update frequently, you do not have to. Update once in two weeks.
I am glad to have new features as soon as they are implemented.

**Reply by valinet** [reply]
First of all, I pushed 3 releases by accident the last hour. I was testing some modifications to my build script. Mistakes happen, want perfection, use products from Microsoft, Google or Apple who have billions of dollars to do a great job... oh, wait... Give me a break...

Second of all, what's the actual problem? You obviously get notifications because you subscribed to them. Just unsubscribe, it's that easy. The app has an update facility, it will notify you about the relevant builds when they are available. Builds and releases are automated. People asked how to get the latest version continuously. That is in reply to that. It also helps me continuously test the builds. I made the change to have only certain commits appear as releases, and the others as per-releases. There's the "beta channel".

**Reply by jed84** [reply]
As said, updates and your hard work are much appreciated. It's an advice, not an attack. You are doing great.
